### DISCUSSION TITLE
Fullscreen window and back

### DISCUSSION THREAD

**Discussion by fsperling** [ianyh/Amethyst#1338] [Ideas]
Hey,

first of all: great work with Amethyst! It helps a lot to make MacOS usability closer to Linuxs.

One thing I'd like to have is after I press `shift`+`alt`+d to fullscreen a window is to
go back to the previous layout when I press it again.

Similar to tmux with zoom in and zoom out.

Maybe I've overlooked something but now when I want to go back I press `shift`+`alt`+`space` and toggle until I find the previous layout.

**Comment by yaio001** [comment]
It would be really a great thing to have.

**Comment by mattviasat** [comment]
👍  I guess the closest workaround is to hotkey the layout, but yes this would be a great feature.

**Comment by ianyh** [comment]
See #1358. The commands for selecting a particular layout will toggle back to the previous layout in 0.20.0, currently in beta as of this comment.

**Comment by ianyh** [comment]
0.20.0 is out now in production, so I'm going to close this one.
